model.py

```diff
@@ -3347,7 +3358,7 @@ def choose_cover(self):
                     if not rep.mirrored_at and not rep.thumbnails:
                         logging.warn(
                             "Best cover for %r (%s) was never mirrored or thumbnailed!",
-                            self.primary_identiifer, 
```

Comment:
I know this is just a spacing change but you've revealed a typo and fixing it would be helpful. This should be `self.primary_identifier`.

Reply:
ok, fixed!